index.js

```diff
@@ -191,26 +260,53 @@ const sReact = (objectData) => {
     console.error("! Can't create reactive object. Need a data object");
     return;
   }
+
+  if ("sReactProperty" in data) {
+    console.error("! sReactProperty const value");
+    return;
+  }
+
+  data["sReactProperty"] = {};
+
   let Core = new Proxy(data, {
     set(target, prop, n) {
+      if (prop === "sReactProperty") {
+        target[prop] = n;
+        return true;
+      }
       if (prop in target) {
         if (target[prop] !== n) {
+          target[prop] = n;
+          target.sReactProperty[prop].value = n;
+          
+          // вот чтоб вот это не делать можно теоретически в sReactProperty подставить type объектам
           const allTag = document.querySelectorAll(`*[tag='${prop}']`);
-
+          const forBlockTags = document.querySelectorAll(`[for-block="${prop}"]`);
+          const forElemsTags = document.querySelectorAll(`[for="${prop}"]`);
           const showTags = document.querySelectorAll(`*[show='${prop}']`);
```

Comment:
yes, it's true, but you can write array attribute `for` and you can write this object on `tag`


for `for` it's one logic, from `tag` it's another logic